Datahub.Core/Data/ResourceProvisioner/CreateResourceData.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+
+namespace Datahub.Core.Data.ResourceProvisioner;
+// ReSharper disable UnusedAutoPropertyAccessor.Global
+public record CreateResourceData
+{
+    
+    // ReSharper disable once MemberCanBePrivate.Global
+    public List<ResourceTemplate> Templates { get; init; }
+    
+
+    public ResourceWorkspace Workspace { get; init; }
+
+    public CreateResourceData(string projectName, string acronym, string sector, string organization)
+    {
+        Templates = new List<ResourceTemplate>() { ResourceTemplate.Default };
+        Workspace = new ResourceWorkspace()
+        {
+            Name = projectName,
+            Acronym = acronym,
+            Organization = new WorkspaceOrganization()
+            {
+                Code = sector,
+                Name = organization,
+            },
+            
+            Users = new List<WorkspaceUser>(),
+        };
+    }
```

Comment:
This is awesome.

I forgot to tell you: since we're switching from the API call to the queue we need to capture the requesting user's email in the message. Could we add a `requestingUserEmail` field?